kbase-extension/static/kbase/js/widgets/narrative_core/upload/stagingAreaViewer.js

```diff
@@ -164,20 +153,7 @@ define([
          */
         setPath: function (path) {
             this.path = path;
-            // factor out the current subdirectory path into its own variable
-            const subpath = path.split('/');
```

Comment:
This was more confusing than anything, and not really helpful. It was used to figure out where files where once selected, but now the entire file path is indexed, so this doesn't matter.